tests/test_extensions.py

```diff
@@ -1,29 +1,32 @@
 from flask_jwt_router._jwt_routes import JwtRoutes
```

Comment:
Improved the tests for extensions .. should only be testing the actual extensions class...